src/puzzles/rush_hour/rush_hour_grid.tscn

```diff
@@ -0,0 +1,52 @@
+[gd_scene load_steps=3 format=3 uid="uid://ch8cal8p7yahi"]
+
+[ext_resource type="PackedScene" uid="uid://dwk61pfqscplh" path="res://src/puzzles/rush_hour/rush_hour_block.tscn" id="1_afvvm"]
+[ext_resource type="Script" path="res://src/puzzles/rush_hour/rush_hour_grid.gd" id="1_wnfx7"]
+
+[node name="RushHourGrid" type="Control"]
+layout_mode = 3
+anchors_preset = 0
+script = ExtResource("1_wnfx7")
+
+[node name="Blocks" type="Control" parent="."]
```

Comment:
Are we going to manually place the blocks for each of the puzzles? Or do we want a system to generate them? Manually place is probably good for our scope, but worth bringing up.

Reply:
Manual for now, i don't know how we'd generate puzzles anyway, you kinda need the human touch

Reply:
I more so meant giving it an array of blocks positions and having it generate the scene from that as opposed to actual proc gen. But I guess it's just a different form of manual input. Just less files I guess?